### DISCUSSION TITLE
Provide debian packages

### DISCUSSION THREAD

**Discussion by igagis** [ChristopherHX/github-act-runner#7] [Ideas]
Have you thought of providing a debian repo with binary packages?
It would be much more convenient installing the runner and updating when new version comes out.
It would be possible to install `systemd` service right away.

**Comment by igagis** [comment]
BTW, I could help with that

**Reply by ChristopherHX** [reply]
Yes please, I never set up or used any debian repository to publish packages.
It seems there is a package helper for go.

**Reply by igagis** [reply]
Ok, I'll investigate it and create a PR when I have something.
Would you like to set up debian repo on your own server? Or I can provide you a repo on mine?

**Reply by ChristopherHX** [reply]
Well I have no real server, I only thought about using github pages.
>  I can provide you a repo on mine?

Would be great

**Reply by igagis** [reply]
deb packages are now being provided!
